turbo.json

```diff
@@ -59,6 +59,7 @@
 		"enforce-size": {
 			"dependsOn": ["build"]
 		},
+		"lint": {},
```

Comment:
could we make `test` depend on `lint` to have it run automatically in all workspaces?